src/pages/Register/RegisterPage/Register.module.scss

```diff
@@ -1,353 +1,504 @@
+                    border-radius: 20px;
+                    width: 35%;
+                    font-size: 1rem;
+                    padding: 2% 2%;
+                    transition: background .2s ease-out;
+
+                    &:hover {
+                        background: darken(#C43388, 10);
                     }
                 }
             }
         }
     }
-}
-
-@media (max-width: 635px) {
-    .signupcomponent {
-        .container {
-            width: 60%;
 
-            .signup {
-                .headlines {
-                    justify-content: center;
-
-                    .headline {
-                        font-size: 18px;
-                    }
-
-                    .subheadline {
-                        font-size: 13px;
-                        text-align: center;
-                    }
-                }
-            }
+    @media screen and (max-width: 992px) {
+        .colone {
+            width: 40%;
         }
-    }
-}
 
-@media (max-width: 530px) {
-    .signupcomponent {
-        .container {
-            width: 70%;
-
-            .signup {
-                .headlines {
-                    justify-content: center;
-
-                    .headline {
-                        font-size: 18px;
-                    }
-
-                    .subheadline {
-                        font-size: 13px;
-                        text-align: center;
-                    }
-                }
-            }
+        .coltwo {
+            width: 60%;
         }
     }
-}
-
-@media (max-width: 455px) {
-    .signupcomponent {
-        .container {
-            width: 80%;
-
-            .signup {
-                .headlines {
-                    justify-content: center;
 
-                    .headline {
-                        font-size: 18px;
-                    }
-
-                    .subheadline {
-                        font-size: 13px;
-                        text-align: center;
-                    }
-                }
-            }
+    @media screen and (max-width: 768px) {
+        .colone {
+            display: none;
         }
-    }
-}
-
-@media (max-width: 399px) {
-    .signupcomponent {
-        .container {
-            width: 90%;
-
-            .signup {
-                .headlines {
-                    justify-content: center;
-
-                    .headline {
-                        font-size: 18px;
-                    }
 
-                    .subheadline {
-                        font-size: 13px;
-                        text-align: center;
-                    }
-                }
-            }
+        .coltwo {
+            width: 100%;
         }
     }
 }
 
-@media (max-width: 355px) {
-    .signupcomponent {
-        .container {
-            width: 100%;
-
-            .signup {
-                .headlines {
-                    justify-content: center;
-
-                    .headline {
-                        font-size: 18px;
-                    }
+// .signupcomponent {
```

Comment:
Why did you left this long chunk of code?

Reply:
Sorry. Thought you'd like to have a look at the previous styles once. Removing them!